### ISSUE TITLE
Add sections

### ISSUE THREAD

**Issue by amrojjeh** [amrojjeh/Arabic#9]

**Comment by amrojjeh** [comment]
This issue can be merged with #6. The syntax could look something like this:
```js
{
    type: "text",
    h1: "This is a new section",
    p1: "One paragraph",
    p2: "Second paragraph",
    h2: "Another section"
}
```

**Comment by amrojjeh** [comment]
Alternatively, we can intentionally limit it to have one header and one paragraph so that none of the quizzes go overboard with explanation:
```js
{
    type: "text",
    h: "This is a new section",
    p: "Only one paragraph",
}
```
